_doc-build-windows/action.yml

```diff
@@ -0,0 +1,285 @@
+# Copyright (C) 2022 - 2024 ANSYS, Inc. and/or its affiliates.
+# SPDX-License-Identifier: MIT
+#
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+name: >
+  Documentation build action (Windows)
+
+description: >
+  Build library documentation using `Sphinx
+  <https://www.sphinx-doc.org/en/master/>`_. The action installs and uses
+  `sphinx-build <https://www.sphinx-doc.org/en/master/man/sphinx-build.html>`_
+  to generate documentation from the source. It requires that all the
+  documentation is contained in the ``doc/`` directory of a project.
+  Documentation dependencies need to be declared in a
+  ``requirements/requirements_doc.txt`` file or in the ``[doc]`` section of the
+  additional dependencies in the ``pyproject.toml`` file. The action locates the
+  ``doc\make.bat`` file and runs the ``doc\make.bat html`` and ``doc\make.bat pdf``
+  commands. If desired, the ``doc\make.bat json`` command can also be executed to
+  generate JSON documentation.
```

Comment:
Just being accurate here. The action does not install Sphinx directly. It assumes that the `sphinx-build` command is available after installing the `requirements.txt` or the `[doc]` dependencies.

If this was the original description, it is a great time to fix it.